### DISCUSSION TITLE
Migrate to PostgresSQL from default SqlLite

### DISCUSSION THREAD

**Discussion by upuldi** [babybuddy/babybuddy#456] [General]
If we can give some scripts to allow users to migrate their choice of database, it would be great.

**Comment by kdrobnyh** [comment]
I believe it's out of scope of the application. You can use [dump](https://www.sqlitetutorial.net/sqlite-dump/) to dump the database and then execute query (`psql -f script.sql` in case of PostgreSQL) to restore it.

**Comment by cdubz** [comment]
Agreed this out of scope -- moving to discussions.
